draft-ietf-masque-h3-datagram.md

```diff
@@ -303,6 +193,10 @@ Capsule Value:
 : The payload of this capsule. Its semantics are determined by the value of the
 Capsule Type field.
 
+Note that use of the Capsule Protocol is not required to use HTTP Datagrams.
+Definitions of new HTTP Methods or of new HTTP Upgrade Tokens can use HTTP
+Datagrams with their own data stream protocol.
+
```

Comment:
This bit belongs up front.

Reply:
Agreed. I refactored the capsule section to contain more information up front.